### DISCUSSION TITLE
Make ports open/closed on demand

### DISCUSSION THREAD

**Discussion by GravySeal** [Bepacom-Raalte/bepacom-HA-Addons#35] [Ideas]
Currently, port 80 and 47808 are always open.
This can cause issues with other add-ons, like Node Red. 
It might be beneficial to be able to enable or disable these ports based on whether you need it.
The integration is currently hard-coded to use port 80, but can be made so the port can be user configurable.

**Comment by Bepacom-Raalte** [comment]
Port 80 is now configurable and can be closed by removing the number in the configuration.
